Pages/Product/product.css

```diff
@@ -0,0 +1,39 @@
+@import url(https://fsharpui.netlify.app/main.css);
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root{
+  --white-color:#fff;
+  --grey-color:#808080;
+
+}
+.navigation {
+  margin: 0rem;
+  width: 100%;
+  background-color: var(--white-color);
+}
+
+.icon-heart,
+.icon-cart {
+  font-size: 3rem;
+}
+
+.filters{
+    flex: 20;

```

Comment:
what does this property do i see no change on removing it?

Reply:
its for flex grow dividing the page into two part